plugins/woocommerce/client/admin/client/marketing/hooks/useInstalledPluginsWithoutChannels.ts

```diff
@@ -7,7 +7,7 @@ import { chain } from 'lodash';
 /**
  * Internal dependencies
  */
-import { STORE_KEY } from '~/marketing/data/constants';
+import { store } from '~/marketing/data';
```

Comment:
Usually, especially in core, it's common to use an alias here

```suggestion
import { store as marketingStore } from '~/marketing/data';
```

You will have to update the references below

Reply:
Thanks. Done in [`25ee49a` (#54485)](https://github.com/woocommerce/woocommerce/pull/54485/commits/25ee49af1d3fd93663a152111787e7421afe6676). I changed it in 3 files with similar occurrences.